website/templates/project/registration_editor_templates.mako

```diff
@@ -154,4 +154,28 @@
     </div>
 </script>
 
+<script>
+ function setAllBoxes(value) {
+     var boxes = document.querySelectorAll('input[type="checkbox"]');
+     $.each(boxes, function(i, box) {
+         this.checked = value;
+     });
+ }
+</script>
+
+<script type="text/html" id="importContributors">
+    <div col-lg-12>
+        <p>Select: <a onClick="setAllBoxes(true)">All</a> | <a onClick="setAllBoxes(false)">None</a></p>
+    </div>
+    <div data-bind="foreach: {data: contributors, as: 'contrib'}">
+        <div class="checkbox">
+            <label>
+                <input type="checkbox" data-bind="value: contrib">
+                <!-- ko text: contrib --> <!-- /ko -->
+            </label>
+        </div>
+    </div>
+    <p>If you would like to add contributors to your OSF project, you can do that on your <a href="${web_url_for('node_contributors', pid=node['root_id'])}">Contributors Page</a> </p>
```

Comment:
Why does this link to the top-level root project, if you are registering a child node?